tests/test_filtsmooth/test_particlefiltsmooth/test_particle_filter.py

```diff
@@ -31,7 +31,7 @@ def test_effective_number_of_events():
 
 @pytest.fixture
 def num_particles():
-    return 4
+    return 10
 
```

Comment:
Also, increasing the number of particles in the PF test, seems to mitigate the flaky-test-issue (at least for the moment?), which further increases the run time.

@pnkraemer Also, I wonder why this emerges now, given that I only refactored, and did not change anything regarding the logic.